dandi/models.py

```diff
@@ -123,21 +125,28 @@ def to_dictrepr(__pydantic_cls__: Type[BaseModel]):
             .replace(__pydantic_cls__.__name__, "dict")
         )
 
+    class Config:
+        @staticmethod
+        def schema_extra(schema: Dict[str, Any]) -> None:
+            schema["title"] = name2title(schema["title"])
+            for name, prop in schema.get("properties", {}).items():
+                if prop.get("title") is None or prop["title"] == name.title():
+                    prop["title"] = name2title(name)
```

Comment:
we need a test for this.  Please add a little test which would pretty much do what `dandi/tools/pubschemata.py` does and ensure that we do get `title` set as expected in at least one record

Reply:
Test added.